scripts/index.js

```diff
@@ -26,28 +27,47 @@ const config =
       ? importCwd('./config/webpack.config.dev')
       : importCwd('react-scripts/config/webpack.config.dev');
 
+const spinner = ora('Update webpack configuration').start();
+
+const chromeExtension = Boolean(backgroundScript) && Boolean(contentScript);
+if (chromeExtension) {
+  spinner.info("Chrome extension watching detected")
+  spinner.info(`    backgoundScript[${backgroundScript}]`);
```

Comment:
I don't know what `background-scripts` and `content-scripts` are and why they are needed. Do you have resources I can look at to better grasps their use?

Reply:
Yeah of course. If you take a look into the general architecture section of a chrome extension: https://developer.chrome.com/extensions/overview#arch you can figure out quickly.